creator-cards/shady-nagy.json

```diff
@@ -4,11 +4,11 @@
   "countryCode": "eg",
   "slogan": "",
   "type": "CREATOR",
-  "bio": "",
+  "bio": "👨‍💻 Software Developer | 🌟 Years of Experience | 🛠️ Proficient in Multiple Languages & Technologies | 🌱 Continuous Learner | 🤝 Team Player | 🔧 Expert in Efficient & Scalable Solutions | 📝 Tech Blogger | 🎁 Open Source Contributor | 💡 Innovator | 🌐 Connect with me!",
   "tags": ".NET, ASP.NET Core",
   "socials": {
     "youtube": "",
-    "linkedin": "",
+    "linkedin": "https://www.linkedin.com/in/shadynagy",
```

Comment:
You can set that directly in your profile

Reply:
done